### DISCUSSION TITLE
Lilygo TTGO LORA T3 V1. 6.1

### DISCUSSION THREAD

**Discussion by UE9AAG** [dl9rdz/rdz_ttgo_sonde#166] [TTGO firmware]
Hi. I used LilyGo TTGO ST32 LORA 433Mhz T3 v1. 6.1 in the MySondyGo project and everything worked well. But I wanted to decode Radium MRZ-N1 probes. I tried to flash your project without rework and it didn't start: black screen and no other activity. Please tell me what could be the matter?

**Comment by dl9rdz** [comment]
How did you flash it? Most common error is to not set the start address to 0x1000 (upper right corner of picture at http://rdzsonde.mooo.com/)

**Reply by UE9AAG** [reply]
Yes, a banal carelessness on my part. I didn't set this address, this field was 0x0000. Thanks for the quick response. Everything started, I will study and test.

**Comment by UE9AAG** [comment]
Hello. Can you add a new weather balloon made in Russia to your project? I have a very simple Arduino sketch on the LoRa LilyGo t3 1.6.1 module with a working decoder for this weather balloon, but I have no knowledge to integrate this code into your project.

**Reply by LukePrior** [reply]
What weather balloon are you speaking about, do you have a GitHub link for your working decoder?

**Reply by bazjo** [reply]
see https://github.com/UE9AAG/Decoder-meteosonde-AK2-02mN

**Reply by bazjo** [reply]
I can remember it no being possible to integrate into auto_rx due to LoRa modulation being used

**Reply by LukePrior** [reply]
Yeah spot on.

**Reply by dl9rdz** [reply]
It certainly could be done with the ESP32

But would need quite some work.
- First remove some features to have space for the LoRa library in the flash
- Then not sure what the transmit cycle of the LoRa TX is, the TTGO software currently can scan multiple frequencies in 1s steps, because a typical transmission cycle of existing sondes is about 1s, possibly that does not make much sense for LoRa transmissions with lower frequency
- Of course if you have a second LoRa module you could scan for "normal" sondes with the one on the TTGO and in parallel continously receive LoRa on the other. In that case, actually things become much easier to integrate, as long as you want to scan only a single LoRa frequency. Basically implement a decoder unit, like the RS41.cpp but much simpler, which in its setup function allows you to change frequence (by manually changing it in the web interface), and a receive() function that does more or less what your arduino sketch main function does, plus copying the recevied data to the SondeInfo struct.

**Reply by ly5p** [reply]
as we all are a little bit maniacs - most of us have few TTGO's :D like me 3 :D it is not a problem to have spare device ant a single freq in it just for decoding first, and later for radiosondy ant/or sondehub. there are many fellows that can test firmware for AK2 sondes :) :)

**Comment by UE9AAG** [comment]
«I'm sorry to be late with the reply. For debugging, I used the Lilugo TTGO v1.6 module. To receive the weather balloon AK2-02Mn, it is required to replace the 32 MHz reference quartz resonator with 30.72 MHz. Without such a refinement, acceptance will not be possible. I did not spoil my TTGO module and soldered the second, modified module in parallel, except for the chip select signal. I connected the chip select signal to IO12. Thus, the circuit diagram completely coincides with the circuit of the Lilugo TTGO v1.6 module, except for the chip selection signal. In addition, simplified home-made versions were assembled, similar to the TTGO module. Unfortunately, my knowledge is not enough to develop and refine the project. The project is not being developed by me and has already been tested by several users in real conditions. Fixed minor errors in the calculation of coordinates, added the functions of decoding the supply voltage, saving information to microSD, changing the frequency via the usb interface. In the near future, this project will be available on github and I will give a link to it.
Anyone interested in receiving and decoding weather balloon signals, please contact me, I will give a link to an active telegram channel dedicated to this hobby.»

  
>Четверг, 19 января 2023, 14:52 +05:00 от ly5p ***@***.***>:
> 
> 
>as we all are a little bit maniacs - most of us have few TTGO's :D like me 3 :D it is not a problem to have spare device ant a single freq in it just for decoding first, and later for radiosondy ant/or sondehub. there are many fellows that can test firmware for AK2 sondes :) :)
>—
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>You are receiving this because you authored the thread. Message ID:  <dl9rdz/rdz_ttgo_sonde/repo-discussions/166/comments/4725960 @ github . com> 
 
 
--
Александр А
Отправлено из Почты  Mail.ru

**Comment by ly5p** [comment]
I would like to be in group. This week i will get quartz from poland and i
am ready to use it in my ttgo ;) please send me telegram link in
***@***.*** thanks.
73! de ly5p

**Comment by UE9AAG** [comment]
https://t.me/+0r-0ojmGdFs3MjM6 понедельник, 23 января 2023г., 21:10 +05:00 от ly5p  ***@***.*** :

>I would like to be in group. This week i will get quartz from poland and i
>am ready to use it in my ttgo ;) please send me telegram link in
>***@***.*** thanks.
>73! de ly5p
>

**Comment by ly5p** [comment]
Спасибо. и где можно скачать последнюю прошивку?
